content/bare/eu/usegalaxy/notices.md

```diff
@@ -5,6 +5,13 @@ For more information, please see our [blog post](https://galaxyproject.org/news/
 
 </div-->
 
+
+
+<div class="alert" style="background: #FFCC000;">
+### **Downtime: Suspicious activity in our cluster**
+
+We recognized suspicious activity in our cluster this morning (Jul 31, 10:50 CEST). To prevent the attack from spreading and protect user data, we took the cluster offline. As a precautionary measure we stopped all running interactive tools and disabled the service until we finished our investigation and took measures. Our team is actively working on resolving the issue, and we will keep you updated on the progress. We apologize for the inconvenience caused.
```

Comment:
```suggestion
We recognized suspicious activity in our cluster this morning (Jul 31, 10:50 CEST). To prevent the attack from spreading and protect user data, we took the services offline. As a precautionary measure, we stopped all running interactive tools and disabled the service until we finished our investigation and took measures. Our team is actively working on resolving the issue, and we will keep you updated on the progress. We apologize for the inconvenience caused.
```